db/data/20220421162907_populate_meeting_options.rb

```diff
@@ -23,6 +23,7 @@ def up
       # GL only options:
       { name: 'glAnyoneCanStart', default_value: 'false' }, # true | false
       { name: 'glAnyoneJoinAsModerator', default_value: 'false' }, # true | false
+      { name: 'glRequireModeratorApproval', default_value: 'false' }, # true | false
```

Comment:
You can't change a migration that's already been run (since it won't run for anyone else). The setting for Moderator Approval alreadys exists - it's called Guest Policy (see v2)